cmd/earthly/flags.go

```diff
@@ -141,11 +141,25 @@ func (app *earthlyApp) rootFlags() []cli.Flag {
 		},
 		&cli.StringFlag{
 			Name:        envFileFlag,
-			EnvVars:     []string{"EARTHLY_ENV_FILE"},
-			Usage:       "Use values from this file as earthly environment variables, buildargs, or secrets",
+			EnvVars:     []string{"EARTHLY_ENV_PATH"},
```

Comment:
not sure if we should keep the old value for backwards compatibility? Since this PR is backwards incompatable, we can rename this too. See my comment below regarding following the same naming scheme.